ci/Dockerfile-envoy-alpine-debug

```diff
@@ -4,6 +4,9 @@ RUN mkdir -p /etc/envoy
 
 ADD build_release/envoy /usr/local/bin/envoy
 ADD configs/google_com_proxy.v2.yaml /etc/envoy/envoy.yaml
+RUN apk update \
```

Comment:
I don't think you need update if you do `add --no-cache` below.

Reply:
seems not. removed